main.pm

```diff
@@ -480,6 +480,9 @@ sub load_zdup_tests() {
 
 sub load_consoletests() {
     if (consolestep_is_applicable) {
+        if (check_var("ADDONS", "rt,ha,geo")) {
```

Comment:
I'm not sure, there is some reason why is this done only with rt,ha,geo, I will ask